draft-ietf-gnap-core-protocol.md

```diff
@@ -125,37 +125,27 @@ Roles are defined by the actions taken and the expectations leveraged
 on the role by the overall protocol. 
 
 Authorization Server (AS)
-: Manages the requested delegations for the RO. 
-    The AS issues tokens and directly delegated information to an instance of the client.
-    The AS is defined by its grant endpoint, a single URL that accepts a POST
-    request with a JSON payload. The AS could also have other endpoints,
-    including interaction endpoints and user code endpoints, and these are
-    introduced to the RC as needed during the delegation process. 
+: server that grants privileges to a particular end-user and that provides them to a client in the form of an access token. 
```

Comment:
We'd have to explain what we mean by "delegated privileges" (that's what the reference to the end-user was trying to circumvent, but maybe it wasn't the best way to do it / I understand what you mean here). 
The rest of your proposal seems fine to be.

Reply:
If we define "privileges" then this is "privileges that have been delegated to the parties named" without need for further definition, I think. It's tricky though.

Reply:
I keep delegated privileges for the commit, will see later if we can find a better proposal

Reply:
I propose not to include "The AS determines which policies apply to any given delegation request." because it's already a sort of requirement, not a definition per say.

Reply:
OK, I see where you're going with the thinner definitions now. We can add in the requirements in a different part.